examples/publisher.nim

```diff
@@ -21,9 +21,9 @@ import
 proc now*(): Timestamp =
   getNanosecondTime(getTime().toUnixFloat())
 
-# An accesible bootstrap node. See wakuv2.prod fleets.status.im
-
+# An accesible bootstrap node. See waku.sandbox fleets.status.im
 
+#TODO: which host is this ENR for?
```

Comment:
Sorry I can't quite understand the question

Reply:
I just checked with https://enr-viewer.com, it is for `node-01.ac-cn-hongkong-c.wakuv2.prod` but peer id changed. Let me replace it with `node-01.ac-cn-hongkong-c.waku.sandbox` one:
```
enr:-QEkuEB3WHNS-xA3RDpfu9A2Qycr3bN3u7VoArMEiDIFZJ66F1EB3d4wxZN1hcdcOX-RfuXB-MQauhJGQbpz3qUofOtLAYJpZIJ2NIJpcIQI2SVcim11bHRpYWRkcnO4bgA0Ni9ub2RlLTAxLmFjLWNuLWhvbmdrb25nLWMud2FrdS5zYW5kYm94LnN0YXR1cy5pbQZ2XwA2Ni9ub2RlLTAxLmFjLWNuLWhvbmdrb25nLWMud2FrdS5zYW5kYm94LnN0YXR1cy5pbQYfQN4DgnJzkwABCAAAAAEAAgADAAQABQAGAAeJc2VjcDI1NmsxoQPK35Nnz0cWUtSAhBp7zvHEhyU_AqeQUlqzLiLxfP2L4oN0Y3CCdl-DdWRwgiMohXdha3UyDw
```